src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AcceptanceTestsInSeparateSubprojectFunctionalSpec.groovy

```diff
@@ -1,16 +1,21 @@
 package info.solidsoft.gradle.pitest.functional
 
+import groovy.transform.CompileDynamic
 import nebula.test.functional.ExecutionResult
 
+@CompileDynamic
 class AcceptanceTestsInSeparateSubprojectFunctionalSpec extends AbstractPitestFunctionalSpec {
 
-    def "should mutate production code in another subproject"() {
+    void "should mutate production code in another subproject"() {
```

Comment:
I believe that some template in Idea uses `def`. Nevertheless, `void` here is move verbose.

Reply:
To be clear - you would like to keep `void` or revert to `def`?

Reply:
Yes, I should be more precise. I meant verbose and meaningful (that method doesn't return anything), not verbose as over-bloated.
Please change to `void`.